src/PagesBody/RolePage/RolePage.js

```diff
@@ -0,0 +1,16 @@
+import { Link } from 'react-router-dom';
+import MainButton from '../../Atoms/MainButton/MainButton';
+import ColumnWrapper from '../../Templates/ColumnWrapper/ColumnWrapper';
+
+const RolePage = () => {
+  return (
+    <ColumnWrapper paddingLeftRight={1}>
+      <h1> Choose Your Role!</h1>
+      <Link to="/contests" style={{ textDecoration: 'none' }}>
+        <MainButton text="Obsługa" primary />
```

Comment:
If this will result in a button nested in anchor tag, we shuold refactor it

Reply:
Its a mock version of this view. Full version will be provided asap.